apis/types/image_progress_info.go

```diff
@@ -0,0 +1,17 @@
+package types
+
+import "time"
+
+// ProgressInfo represents the status of downloading image.
+type ProgressInfo struct {
```

Comment:
In dir /apis/types, all structs are auto generated by apis/swagger.yml. So if you could wanna add a struct here, you must edit the swagger.yml and auto generate this struct. Thanks. And you can refer to more details here at https://github.com/alibaba/pouch/blob/master/apis/README.md